app/controllers/concerns/state_file/state_file_controller_concern.rb

```diff
@@ -3,26 +3,7 @@ module StateFileControllerConcern
     extend ActiveSupport::Concern
 
     included do
-      helper_method :current_intake, :current_state_code, :current_state_name, :current_tax_year
-    end
-
-    def current_intake
-      StateFile::StateInformationService.active_state_codes
-                                        .lazy
-                                        .map{ |c| send("current_state_file_#{c}_intake".to_sym) }
-                                        .find(&:itself)
-    end
-
-    def current_state_code
-      if current_intake
-        current_intake.state_code
-      else
-        params[:us_state]
-      end
-    end
-
-    def current_state_name
-      StateFile::StateInformationService.state_name(current_state_code)
+      helper_method :current_tax_year
```

Comment:
should this just be rolled in to StateFileIntakeConcern? having trouble thinking through whether that makes sense